### PR TITLE
PTL-1668 - adding openfin notifications listener doc

### PR DESCRIPTION
**Pull request by patrickoneill-genesis**
ATTENTION! NEW DOCS STRUCTURE
FOR THE NEXT THREE WEEKS, ONLY BRANCH FROM new-docs-structure
OTHERWISE, WAIT UNTIL THE NEW STRUCTURE HAS BEEN RELEASED

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you done a trial build to check all new or changed links?

Is there anything else you would like us to know?

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+    <TabItem value="react">
+        Declaration
+        ```html
+        <rapid-openfin-notification-listener></rapid-openfin-notification-listener>
+        ```
+
+        Usage
+        ```typescript
+        export function MyComponent() {
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx:76]: Same comment as other PR regarding to either set the properties directly, or you need to declare the ref

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+        <th>Type</th>
+        <th>Use</th>
+        <th>Example</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>toastButtons</td>
+        <td>
+            NotificationsPlatform <!-- To Do add link-->
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx:206]: You've added the fdc3 docs right? So can't you do the link already?

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+        </td>
+        <td>An array of configs for showing buttons. Buttons appear in notifications when the <code>condition</code> callback returns <code>true</code>. The action callback is called when the corresponding button is clicked</td>
+        <td>
+            ```typescript
+            <rapid-openfin-notification-listener :toastButtons=${)_ => [
```
Comment (matteematt) [inline docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx:211]:
```suggestion
            <rapid-openfin-notification-listener :toastButtons=${() => [

```

**Review by matteematt — COMMENTED** [review]
Similar comments to the other PR

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx (hunk 2 above)
Comment (patrickoneill-genesis) [reply docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx]: Link updated

File: docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx (hunk 3 above)
Comment (patrickoneill-genesis) [reply docs/001_develop/03_client-capabilities/021_desktop-interoperability/007_openfin-notifications-listener.mdx]: Fixed

**Review by matteematt — APPROVED** [review]

**Pull request merged**